packages/angular_devkit/core/node/resolve.ts

```diff
@@ -10,6 +10,17 @@ import * as fs from 'fs';
 import * as path from 'path';
 import { isFile } from './fs';
 
+// Parse collection from command line args
+const argvSchematicPath = process.argv[2];
+const argvCollection = argvSchematicPath.substr(0, argvSchematicPath.indexOf(':'));
```

Comment:
The `@angular-devkit/core` package is a library.  It shouldn't assume the presence of command line arguments.

Reply:
I will rethink this.  If possible, please leave the PR open and I will try to push the changes in a timely fashion. Thanks @clydin .